### PR TITLE
Explicitly Include Affinity & Toleration Options in PostgresSQL

### PR DESCRIPTION
**Pull request by Shomu-Maersk**
Thanks for the wonderful helm solution so that anyone can deploy Dify in Kubernetes. That being said I have identified an issue where the repository lacked example values for tolerations and affinity for the Postgres (master & replicas) deployment. This PR includes an updated values.yaml file with a commented example showcasing how to configure these settings for the Postgres database. Best
Shomu

### CODE REVIEW

**Comment by BorisPolonsky** [comment]
Thanks for your contribution. `Postgresql` were from [bitnami/charts](https://github.com/bitnami/charts/blob/main/bitnami/postgresql/values.yaml). We shall check if this modification complies with `.Values.yaml` defined by `bitnami`

**Comment by Shomu-Maersk** [comment]
PR updated as per comments.

**Comment by BorisPolonsky** [comment]
Hello there. The comments [here](https://github.com/Shomu-Maersk/dify-helm/blob/2925623ae174a8996b5bb6273fc87e266f747bbd/charts/dify/values.yaml#L316-L325) suggests `primary` but indeed they are for `readReplicas`. Please update them as follow:
```
  affinity: {}
  ## @param readReplicas.nodeSelector Node labels for PostgreSQL read only pods assignment
  ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/
  ##
  tolerations: []
  ## @param readReplicas.topologySpreadConstraints Topology Spread Constraints for pod assignment spread across your cluster among failure-domains. Evaluated as a template
  ## Ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/#spread-constraints-for-pods
  ##
```

File: charts/dify/values.yaml [hunk 1]
```diff
@@ -302,6 +313,17 @@ postgresql:
         - ReadWriteOnce
       size: 8Gi
 
+    ## @param readReplicas.affinity Affinity for PostgreSQL read only pods assignment
+    ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+    ## Note: primary.podAffinityPreset, primary.podAntiAffinityPreset, and primary.nodeAffinityPreset will be ignored when it's set
+    ##
+    affinity: {}
+
+    ## @param primary.tolerations Tolerations for PostgreSQL primary pods assignment
```
Comment (BorisPolonsky) [inline charts/dify/values.yaml:322]: Comment here suggests `primary` but indeed they are for `readReplicas`. Please update them as follow:
```
  tolerations: []
  ## @param readReplicas.topologySpreadConstraints Topology Spread Constraints for pod assignment spread across your cluster among failure-domains. Evaluated as a template
  ## Ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/#spread-constraints-for-pods
  ##
```

**Comment by Shomu-Maersk** [comment]
Hi @BorisPolonsky, updated the PR with readReplicas.tolerations. 
Best 
Shomu

**Pull request merged**